modules/apps/object/object-js-components-web/src/main/resources/META-INF/resources/utils/string.ts

```diff
@@ -12,13 +12,17 @@ const userLanguageId = Liferay.ThemeDisplay.getLanguageId();
 
```

Comment:
Found a corner case in Objects & Notifications modules.
Both modules use widely this helper function to get the localized label from an object.
They also use it as part of a FDS cellrenderer, expecting an object to extract the correct key.